xml/docu_styleguide.language.xml

```diff
@@ -83,55 +83,15 @@
    capitalize the first word of a sentence and proper names.
   </para>
   <para>
-   Use title-style capitalization for all types of headings and titles,
-   including admonition, figure, and table titles. This style is explained in
-   <citetitle>The Chicago Manual of Style</citetitle> 8.167. A simplified
-   version of these rules is below:
+   Use sentence-style capitalization for all types of headings and titles,
+   including admonition, figure, and table titles. This means that only
+   the first word of any title should be capitalized. For example:
+   "Chapter one".
```

Comment:
This is not a good example, as something like "Chapter one" is usually just an auto-generated string. Feel free to pull any of the titles you have converted in SES docs as an example.